### PR TITLE
Update CI; Swift 5.2 is now minimum; Update dependencies

### PR DESCRIPTION
**Pull request by dannys42**

### CODE REVIEW

**Review by LowAmmo — APPROVED** [review]

**Review by LowAmmo — APPROVED** [review]

File: SwiftJWT.podspec [hunk 1]
```diff
@@ -1,21 +1,21 @@
...
   s.ios.deployment_target = "11.0"
   s.tvos.deployment_target = "11.0"
   s.watchos.deployment_target = "4.0"
   s.source       = { :git => "https://github.com/Kitura/Swift-JWT.git", :tag => s.version }
   s.source_files  = "Sources/**/*.swift"
   s.dependency 'BlueRSA', '~> 1.0.200'
-  s.dependency 'BlueECC', '~> 1.1.0'
-  s.dependency 'LoggerAPI', '~> 1.7.0'
-  s.dependency 'KituraContracts', '~> 1.2.200'
   s.dependency 'BlueCryptor', '~> 2.0.1'
```
Comment (LowAmmo) [inline SwiftJWT.podspec:17]: I submitted a PR for BlueCryptor that would clean up the "cryptographically broken" MD2, MD4, & MD5 options...so...might end up needing to up this to like a `~> 3.0`...

**Review by LowAmmo — APPROVED** [review]

File: SwiftJWT.podspec (hunk 1 above)
Comment (dannys42) [reply SwiftJWT.podspec:17]: I'll address in a future PR.

**Comment by LowAmmo** [comment]
@dannys42  Thanks for updating!

Any chance you can publish the new 4.0.0 version to cocoapods? 😄

**Comment by dannys42** [comment]
@LowAmmo Looks like BlueRSA and BlueECC have to be updated and pushed as well.  I'll see what I can do.

**Comment by LowAmmo** [comment]
Awesome! Thanks for the attention and quick responses, @dannys42 !

You are a gentleman and a scholar

**Pull request merged**